src/components/new-feature.js

```diff
@@ -0,0 +1,21 @@
+import React from "react"
+import { Link } from 'gatsby'
```

Comment:
If you import link from `'gatsby-plugin-intl'`, I think it will automatically handle the path of the link based on the language selected.

Reply:
Wow, thank you!